app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.java

```diff
@@ -171,7 +173,7 @@ static void addEventMetadata(Event event) {
         /*
          * When QUEUE.size() exceeds this value TIMER becomes non-interruptable.
          */
-        private static final int MAX_QUEUE_SIZE = 128;
+        private static final int MAX_QUEUE_SIZE = WikipediaApp.getInstance().isOnline() ? 10 : 128;
```

Comment:
Since this is `static` and `final`, won't it be initialized once and stay that way forever?

Reply:
whoops!